appserver/web/war-util/src/main/java/org/glassfish/web/loader/WebappClassLoader.java

```diff
@@ -652,18 +635,14 @@ public void setWorkDir(File workDir) {
 
     public void setUseMyFaces(boolean useMyFaces) {
         this.useMyFaces = useMyFaces;
-        if (useMyFaces) {
```

Comment:
@dmatej 
Without this clause, users cannot override the faces package even if they enable useMyFaces.